src/test/java/com/google/api/codegen/testdata/php_main_library.baseline

```diff
@@ -1207,7 +1213,7 @@ class LibraryServiceApi
      * try {
      *     $libraryServiceApi = new LibraryServiceApi();
      *     $formattedName = LibraryServiceApi::formatBookName("[SHELF]", "[BOOK]");
-     *     $indexName = "";
+     *     $indexName = "default index";
```

Comment:
I believe this feature was only implemented in Java (until now). See for example here:
https://github.com/googleapis/toolkit/blob/master/src/test/java/com/google/api/codegen/testdata/python_main_library.baseline#L856